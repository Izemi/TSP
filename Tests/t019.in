AAA,0.0,0.0
BBB,0.0,2.0
CCC,0.0,5.0
DDD,-3.0,3.0
EEE,-2.0,-10.0
