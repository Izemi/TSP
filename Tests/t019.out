-insert   :       721.47 HVN BDL ALB ORH MHT PVD HVN
-given    :       805.78 HVN ALB MHT BDL ORH PVD HVN
-nearest  :       734.16 HVN ALB MHT PVD ORH BDL HVN
